server/src/resolvers/employee.resolver.ts

```diff
@@ -131,6 +131,30 @@ class EmployeeResolver {
     await foundOrg.save();
     return saved;
   }
+
+  @Mutation(() => DealershipUser)
+  async updateEmployeeById(
+    @Ctx() { req }: ServerContext,
+    @Arg("updatedUser", () => UpdateUser) updatedUser: UpdateUser
+  ): Promise<DealershipUser | null> {
+    console.log("req session", req.session);
+    if (!req.session.userId) {
+      throw new ApolloError("Not Authorized");
+    }
+    const user = await DealershipUser.findOne(req.session.userId, {
+      relations: ["dealershipOrganization", "dealershipOrganization.employees"],
+    });
+
+    for (const key in updatedUser) {
+      if (Object.prototype.hasOwnProperty.call(updatedUser, key)) {
+        if (!!updatedUser[key]) {
+          user[key] = makeDbSearchable(updatedUser[key] as string);
+        }
+      }
+    }
```

Comment:
Analyze this function and see if there is any way we can extract it. It would be nice to use it in other areas.

Reply:
Doesn't need to be completed for merge.